cmdLine/src/main/scala/topgun/cmdline/RecordedEventLabelMapping.scala

```diff
@@ -0,0 +1,13 @@
+package topgun.cmdline
+
+// TODO find reference values inside JMC/JFR and replace with constants.
+//  RecrodedEvent.AllocatedInNewTLAB
+object RecordedEventLabelMapping {
+  val AllocationInNewTLAB = "Allocation in new TLAB"
+  val AllocationOutsideTlab = "Allocation outside TLAB"
+  val MethodProfilingSample = "Method Profiling Sample"
+  val MethodProfilingSampleNative = "Method Profiling Sample Native"
+  val InitialSystemProperty = "Initial System Property"
+  val RecordingSetting = "Recording Setting"
+  val OSInformation = "OS Information"
```

Comment:
I think that its better to switch on `(event: Event) => e.getEventType().getType().getName()`, which has ids like `jdk.ObjectAllocationInNewTLAB`. (full list: https://github.com/openjdk/jdk/blob/master/src/hotspot/share/jfr/metadata/metadata.xml)